test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-product_secondary_unit @ git+https://github.com/OCA/product-attribute.git/refs/pull/1246/head#subdirectory=setup/product_secondary_unit
```

Comment:
```suggestion
odoo-addon-product_secondary_unit @ git+https://github.com/OCA/product-attribute.git@refs/pull/1246/head#subdirectory=setup/product_secondary_unit
```

Reply:
upps!!! The doc is wrong.
`odoo-addon-<module_name> @ git+https://github.com/OCA/<repository>.git/refs/pull/<PR_number>/head#subdirectory=setup/<module_name>`

I will try it!! Thanks @chienandalu

Reply:
I have just fixed the documentation. The example were good though.